app/views/newflow/login_signup/_password_show_hide.html.erb

```diff
@@ -0,0 +1,19 @@
+<span id="password-show-hide-button">
+    <span class="toggle-show-hide">SHOW</span>
+    <span class="toggle-show-hide" style="display: none">HIDE</span>
+</span>
+
+<script type="text/javascript">
+  $(window).ready(function() {
+      var pwd_input = document.getElementById('signup_password');
+      var pwd_checkmark = document.getElementById('password-requirements-checkmark');
+
+      pwd_input.oninput = function(event){
```

Comment:
where is this invoked?  just not clear why it's stored here on a pwd_input as a variable.

Reply:
That's the way it works? :) The function gets called `oninput`